README.md

```diff
@@ -8,20 +9,97 @@
 FRIEND is an IDA plugin created to improve disassembly and bring register/instruction documentation right into IDA View.
 
 ### 1. Improved processor modules using third party libraries (like Capstone)   
+
 ![](./Resources/screenshots/Capstone.png)
 
 ### 2. Hints for instructions and registers in IDA View   
+
 ![](./Resources/screenshots/BCC.png)
 ![](./Resources/screenshots/DSB_inst.png)
 ![](./Resources/screenshots/DSB_reg.png)
 ![](./Resources/screenshots/UMADDL.png)
 
 ### 3. Ability to show external reference for highlighted item in a browser    
+
 ![](./Resources/screenshots/ShowDoc.png)
 
 ### 4. Ability to pick only elements you are interested in
+
 ![](./Resources/screenshots/Settings.png)
 
+## How to build
+
+### Preparing the build environment
+
+To build the IDA plugin, there are few dependencies to satisfy:
+
+* [CMake](https://cmake.org/download/) 3.3 or higher
+* GCC or Clang on Linux/macOS. On Windows, use the
+  Visual Studio 2015.
+* Git
+* IDA SDK (unpack into ``idasdk``)
+
+Unzip the contents of the IDA SDK into `idasdk`. On Linux or MacOS, one can use the following commands:
+
+```sh
+$ unzip /path/to/idasdk69.zip -d idasdk
+$ mv idasdk/idasdk69/* idasdk
+$ rm -r idasdk/idasdk69
+```
+
+### Linux
+
+Use ``cmake`` to prepare the build environment and run ``make`` to build the plugins:
+
+````sh
+$ mkdir _build
+$ cd _build
+$ cmake ..
+$ make
+```
+
+### MacOS
+
+Use ```cmake to prepare the build environment and run ``make`` to build the plugins:
```

Comment:
```
-Use ```cmake to prepare the build environment and run ``make`` to build the plugins:
+Use ``cmake`` to prepare the build environment and run ``make`` to build the plugins:
```